portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,50 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Angello's Portfolio</title>
+    <!-- My stylesheets -->
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container">
+      <div class="nav-wrapper">
+        <div class="nav-left">
+          <div class="nav-link">
+            <a href="index.html">Home</a>
+          </div>
+          <div class="nav-link">
+            <a href="about.html">About me</a>
+          </div>
+          <div class="nav-link">
+            <a href="projects.html">Projects</a>
+          </div>
+        </div>
+
+        <div class="nav-right">
+          <div id="nav-name">
+            Angello Parrolivelli
+          </div>
+        </div>
+      </div>
+
+      <div id="content">
+        <div class="header">
+          <h1 class="g-green">Hi there!</h1>
+          <div id="intro">My name's <span class="g-blue">Angello</span>, and this is the portfolio I made in Google's SPS program!</div>
```

Comment:
Google style guide for HTML recommends breaking long lines ([Style guide](https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping)) . It is not required unlike other languages like c++ or java, but still useful. The main benefit is to avoid horizontal scrolling when reading the code.

Reply:
Got it, I will take a look at the style guide and adjust this line and any others in a commit asap.